mstickerlib/src/image.rs

```diff
@@ -41,6 +41,13 @@ impl Image {
 		))
 	}
 
+	pub async fn convert_tgs_if_some(self, animation_format: Option<AnimationFormat>) -> anyhow::Result<Self> {
```

Comment:
is this really necessary as a public function?

Reply:
Probably not. I think the half of everything which is currently pub should not be pub. I should review this everywhere, before the next release.